tests/api_fastapi/core_api/routes/public/test_assets.py

```diff
@@ -523,6 +523,42 @@ def test_should_respond_404(self, test_client):
         assert response.json()["detail"] == "Queue event with dag_id: `not_exists` was not found"
 
 
+class TestGetDagAssetQueuedEvent(TestQueuedEventEndpoint):
+    @pytest.mark.usefixtures("time_freezer")
+    def test_should_respond_200(self, test_client, session, create_dummy_dag):
+        dag, _ = create_dummy_dag()
+        dag_id = dag.dag_id
+        self.create_assets(session=session, num=1)
+        asset_id = 1
+        self._create_asset_dag_run_queues(dag_id, asset_id, session)
+        asset_uri = "s3://bucket/key/1"
+
+        response = test_client.get(
+            f"/public/dags/{dag_id}/assets/queuedEvent/{asset_uri}",
```

Comment:
not related to this PR. We should keep this PR as it is.


but would like to confirm wit h @uranusjr, we probably would like to use the name for airflow 3?

Reply:
@Lee-W resolving this PR for the sake of merging it, please create an issue if this is needed and you can assign it to me, we can prioritise and work on it

CC @pierrejeambrun @dstandish